src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -228,17 +230,14 @@ export const PublishingAccordion = ({
                     : ''
                 : t('registration.public_page.tasks_panel.registration_will_soon_be_published')}
             </Typography>
-            <LoadingButton
-              variant="contained"
-              color="info"
-              size="small"
-              loading={isLoadingData}
-              fullWidth
-              onClick={refetchData}
-              startIcon={<RefreshIcon />}
-              data-testid={dataTestId.registrationLandingPage.tasksPanel.refreshPublishingRequestButton}>
-              {t('registration.public_page.tasks_panel.reload')}
-            </LoadingButton>
+            <RefreshPublishingRequestButton refetchData={refetchData} isLoadingData={isLoadingData} />
+          </>
+        )}
+
+        {isDeletedRegistration && isWaitingForFileDeletion && (
+          <>
+            <Typography gutterBottom>{t('registration.public_page.tasks_panel.files_will_soon_be_deleted')}</Typography>
```

Comment:
Husker det har vært mye knoting med elementene i denne boksen, da den er bygd opp på en litt teit måte hvor elementene selv må definerer margin til andre elementer i stedet for å bare bruke en enkel flex.. Så pass på at teksten her ikke kommer helt inntil noe over, da jeg ser at denne har `gutterBottom` mens lignende blokk over har `sx={{ my: '1rem' }}`. Virker naturlig at de skulle vært like 🤔